Source/Scene/ShadowMap.js

```diff
@@ -185,8 +185,9 @@ define([
         this._needsUpdate = true;
 
         // In IE11 polygon offset is not functional.
+        // TODO : Also disabled for Chrome temporarily. Re-enable once https://groups.google.com/forum/#!topic/webgl-dev-list/E1dAG65QBhg is resolved.
```

Comment:
Ok, the new issue is here: https://github.com/AnalyticalGraphicsInc/cesium/issues/4560